editor/src/clj/editor/asset_browser.clj

```diff
@@ -361,9 +361,10 @@
   (assert (and new-name (not (string/blank? new-name))))
   (let [workspace (resource/workspace resource)
         src-file (io/file resource)
-        ^File dest-file (File. (.getParent src-file) new-name)
-        dest-proj-path (resource/file->proj-path (workspace/project-path workspace) dest-file)]
-    (when-not (resource-watch/reserved-proj-path? dest-proj-path)
+        dest-file (File. (.getParent src-file) new-name)
+        project-directory-file (workspace/project-path workspace)
+        dest-proj-path (resource/file->proj-path project-directory-file dest-file)]
+    (when-not (resource-watch/reserved-proj-path? project-directory-file dest-proj-path)
```

Comment:
I wouldn't have anticipated any changes to "asset_browser.clj"?
Does it mean I cannot find/open files from those folders? (E.g. documentation etc)

Reply:
The excluded folders will be just that, completely excluded from the project. They will not be loaded on project startup as part of the resource graph. This also means they will not be included in the asset browser.

I'm using Atom and when I open a folder containing a git repo I do not see any files matching the patterns in .gitignore. The same applies here with .defignore.